### PR TITLE
Support Unreal Engine 5.5

### PR DESCRIPTION
**Pull request by pdxmholmes**
Make the small set of changes needed to support Unreal 5.5.

### CODE REVIEW

File: tools/replace_engine_version.sh
Comment (nutti) [inline tools/replace_engine_version.sh:1]: Below code needs to be changed.

```
for file in `find ${source_dir} -name "*.uplugin"`; do
    sed -i -e "s/\"EngineVersion\": \"5.4.0\",/\"EngineVersion\": \"${engine_version}\",/g" ${file}
    echo "Replaced engine version in ${file}"
done
```

**Review by nutti — CHANGES_REQUESTED** [review]
Thank you @pdxmholmes !

We are now confirming this patch works fine.
I request a small change.

File: tools/replace_engine_version.sh
Comment (pdxmholmes) [reply tools/replace_engine_version.sh:1]: I'll take care of this tonight.

**Comment by nutti** [comment]
@pdxmholmes 

Sorry, I deleted the commits by mistake.
Could you apply these changes to your main branch?

https://github.com/colory-games/UEPlugin-AccessVariableByName/tree/pdxmholmes-pr-fix

**Comment by nutti** [comment]
This PR is now merged by https://github.com/colory-games/UEPlugin-AccessVariableByName/commit/bf5e641eb0529156560dbb5421ee1f442be6a6e9 which includes CI fixes.
Thank you @pdxmholmes for your contribution!

**Comment by pdxmholmes** [comment]
> @pdxmholmes
> 
> Sorry, I deleted the commits by mistake. Could you apply these changes to your main branch?
> 
> https://github.com/colory-games/UEPlugin-AccessVariableByName/tree/pdxmholmes-pr-fix

Sorry I missed this, was away for the Christmas holidays. Seems like you got it working and merged!

**Comment by nutti** [comment]
@pdxmholmes 

No prob. Thank you for your contribution!

**Pull request closed without merging**